### PR TITLE
added screenshots viewer

### PR DESCRIPTION
**Pull request by introkun**

### CODE REVIEW

**Review by Aemiii91 — APPROVED** [review]

File: static/packages/App/Screenshots (View screenshots)/App/Screenshots_Viewer/launch.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+cd /mnt/SDCARD/App/Screenshots_Viewer
+/mnt/SDCARD/.tmp_update/bin/infoPanel -d /mnt/SDCARD/Screenshots --show-theme-controls
```
Comment (Aemiii91) [inline static/packages/App/Screenshots (View screenshots)/App/Screenshots_Viewer/launch.sh:3]: Showing the theme controls on top of screenshots is not desired. You either need to resize the screenshots or implement a toggle on Y (the same as GS).

File: static/packages/App/Screenshots (View screenshots)/App/Screenshots_Viewer/config.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+"label":"Screenshots",
+"icon":"icon.png",
+"launch":"launch.sh",
+"description":"View screenshots"
```
Comment (Aemiii91) [inline static/packages/App/Screenshots (View screenshots)/App/Screenshots_Viewer/config.json:5]: I would like it to be called label: "Gallery", description: "Screenshot viewer", and folder name in packages should be "Gallery (Screenshot viewer)".

**Review by Aemiii91 — CHANGES_REQUESTED** [review]

File: static/packages/App/Screenshots (View screenshots)/App/Screenshots_Viewer/config.json (hunk 2 above)
Comment (introkun) [reply static/packages/App/Screenshots (View screenshots)/App/Screenshots_Viewer/config.json]: Fixed in [904982f](https://github.com/OnionUI/Onion/pull/115/commits/904982f375e1d2473668a44c2b9340e8569519e6)

File: static/packages/App/Screenshots (View screenshots)/App/Screenshots_Viewer/launch.sh (hunk 1 above)
Comment (introkun) [reply static/packages/App/Screenshots (View screenshots)/App/Screenshots_Viewer/launch.sh]: Fixed by [645d6d6](https://github.com/OnionUI/Onion/pull/115/commits/645d6d6d5d28f73b0b5dfd75b30cebd9b6c9048c)

**Review by Aemiii91 — APPROVED** [review]

**Comment by introkun** [comment]
Fix will be provided in #124

**Pull request closed without merging**